service/app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ActiveRecord::Base
+	include ApplicationModel
+
+	has_one   :user_account
```

Comment:
__[rubocop]__
* Tab detected. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)
* Put one space between the method name and the first argument.